lib/elements/satin_column.py

```diff
@@ -513,6 +514,11 @@ def rail_indices(self):
             indices_by_length = sorted(list(range(num_paths)), key=lambda index: paths[index].length, reverse=True)
             return indices_by_length[:2]
 
+    @property
```

Comment:
Probably worth caching, since it gets called in every `get_split_points()` call.